yarn.lock

```diff
@@ -17099,7 +17099,7 @@ uuid@^3.0.0, uuid@^3.3.2, uuid@^3.3.3:
   resolved "https://registry.npmjs.org/uuid/-/uuid-3.4.0.tgz#b23e4358afa8a202fe7a100af1f5f883f02007ee"
   integrity sha512-HjSDRw6gZE5JMggctHBcjVak08+KEVhSIiDzFnT9S9aegmp85S/bReBVTb4QTFaRNptJ9kuYaNhnbNEOkbKb/A==
 
-uuid@^8.3.2:
+uuid@^8.0.0, uuid@^8.3.2:
```

Comment:
Is this intentional? Seems like we shouldn't need to pull in an older version.

Reply:
Good catch. I assumed yarn would've resolved to the existing one installed in the repo

Reply:
Looks like this is still here?